products/appengine/inspec.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Inspec::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  FirewallRule: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: true
+  StandardAppVersion: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: false
+    properties:
+      id: !ruby/object:Overrides::Inspec::PropertyOverride
+        name: "version_id"
+      runtimeApiVersion: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true  
+      handlers: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true  
+      libraries: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true  
+      envVariables: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true
+      deployment: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true
+      entrypoint: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true
+      service: !ruby/object:Overrides::Inspec::PropertyOverride
+        exclude: true    
+  Service: !ruby/object:Overrides::Inspec::ResourceOverride
+    exclude: true  
+  
```

Comment:
Looks like we need an entry for 
```
  DomainMapping: !ruby/object:Overrides::Inspec::ResourceOverride
    exclude: true
```

It has been added in master recently

Reply:
Added